language/move-lang/functional-tests/tests/libra/libra_account/add_currency.move

```diff
@@ -1,14 +1,79 @@
+//! account: validator, 1000000, 0, validator
 //! account: vasp, 0,0, address
 //! account: child, 0,0, address
+//! account: vivian, 0, 0, address
+//! account: otto, 0, 0, address
+
+
+// BEGIN: registration of a currency
```

Comment:
:( If only there was a way to invoke some test helper to avoid replicating this code

Reply:
I wish too 😢  But! I think in the new and improved testing framework that Victor is building this should be possible :)